docker-bake.hcl

```diff
@@ -142,9 +141,12 @@ target "test-fixture" {
     dockerfile = "test/fixtures/${f}.yml"
 
     args = {
-        "BUILDKIT_SYNTAX" = FRONTEND_REF
+        "BUILDKIT_SYNTAX" = "dalec_frontend"
         "DALEC_DISABLE_DIFF_MERGE" = DALEC_DISABLE_DIFF_MERGE
     }
+    contexts = {
```

Comment:
Does this mean `target:frontend` needs to be a build context?

Reply:
This meas the bake target *is* passing the frontend as a build context and then the build-arg tells it to read the frontend from the context.